src/notification/notification.service.ts

```diff
@@ -15,8 +16,14 @@ export default class NotificationService {
         senderUserID: string,
         receiverUserID: string
     ): Promise<void> {
+        const destination: string | undefined = !this.isValidObjectId(
```

Comment:
Why not just default it to an empty string?

Reply:
Same but ok